server/data/data-library/src/test/java/org/apache/james/domainlist/lib/AbstractDomainListTest.java

```diff
@@ -169,7 +173,7 @@ private void deleteWithoutError(String domain) {
         try {
             domainList.removeDomain(domain);
         } catch(DomainListException e) {
-
+            LOGGER.info("Ignored error", e);
```

Comment:
No here the error is intentionally swallowed. I added a log message to not loose track. I will not break this weird behaviour.